dataduct/steps/scripts/sql_runner.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
```

Comment:
no as this is shared across multiple sql abstractions that create and run some sql